cmd/teleproxy/teleproxy_test.go

```diff
@@ -98,20 +103,26 @@ func teleproxyCluster() {
 var smoke = testprocess.MakeSudo(teleproxyCluster)
 
 func TestSmoke(t *testing.T) {
+	if noDocker != nil {
+		t.Skip(noDocker)
+	}
```

Comment:
NB: I added this, because without k3s, anything using `{{image}}` won't work on macOS CI.